models/cow_protocol/tx_hash_labels/early_investment/ethereum/cow_protocol_tx_hash_labels_early_investment_ethereum.sql

```diff
@@ -0,0 +1,54 @@
+{{
+    config(
+        alias='tx_hash_labels_early_investment_ethereum',
+    )
+}}
+
+with
+  project_starts as (
+    select
+        token_bought_address,
+        min(block_date) as project_start
+    from (
+        select token_bought_address, block_date
+        from {{ ref('dex_aggregator_trades') }}
+        where blockchain = 'ethereum'
+        UNION ALL
+        select token_bought_address, block_date
+        from {{ ref('dex_trades') }}
+        where blockchain = 'ethereum'
+    )
+    group by
+        token_bought_address
+
+ ),
+
+ early_investment_trades as (
+    select
+        tx_hash
+    from (
+        select tx_hash, block_date, token_bought_address
+        from {{ ref('dex_aggregator_trades') }}
+        where blockchain = 'ethereum'
+        UNION ALL
+        select tx_hash, block_date, token_bought_address
+        from {{ ref('dex_trades') }}
+        where blockchain = 'ethereum'
+    ) t join project_starts p on t.token_bought_address = p.token_bought_address
+    where
+        date_diff(t.block_date, p.project_start) <= 30
```

Comment:
Maybe as a comment that we have deemed 30 days to be considered an early investment.